File: TestApplication/app/src/main/java/com/example/testapplication/MainActivity.java
package com.example.testapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Animal a1,a2,a3;
        a1=new Animal();
        a2=new Dog();
        a3=new BabyDog();
        a1.eat();
        a2.eat();
        a3.eat();

    }
}

File: Design/app/src/main/java/com/example/design/MainActivity.kt
package com.example.design

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.design.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        setUpData()
    }

    private fun setUpData() {
        binding.btnOneCloudDesign.setOnClickListener {
            val accountSecurityPage = Intent(this, OneCloudAccountSecurity::class.java)
            startActivity(accountSecurityPage)
        }
        binding.btnProfilePage.setOnClickListener {
            val userProfilePage = Intent(this, UserProfileActivity::class.java)
            startActivity(userProfilePage)
        }
        binding.btnMedicinePage.setOnClickListener {
            val medicinePage = Intent(this, CoughMedicineActivity::class.java)
            startActivity(medicinePage)
        }
    }
}